Sources/SwiftCrossUI/Views/Menu.swift

```diff
@@ -71,27 +74,45 @@ public struct Menu: TypeSafeView {
         size.x = buttonWidth ?? size.x
 
         let content = resolve().content
-        backend.updateButton(
-            widget,
-            label: label,
-            action: {
-                let menu = backend.createPopoverMenu()
+        switch backend.menuImplementationStyle {
+            case .dynamicPopover:
+                backend.updateButton(
+                    widget,
+                    label: label,
+                    action: {
+                        let menu = backend.createPopoverMenu()
+                        children.menu = menu
+                        backend.updatePopoverMenu(
+                            menu,
+                            content: content,
+                            environment: environment
+                        )
+                        backend.showPopoverMenu(menu, at: SIMD2(0, size.y + 2), relativeTo: widget)
```

Comment:
Line wrap it like this (apologies that the formatter config doesn't do this automatically);

```swift
function(
    arg1,
    arg2,
    arg3
) {
    ...
}
```